### PR TITLE
[MOSIP-23969] modified regclient install.sh ... added openssl creation as preinstall job

### PR DESCRIPTION
**Pull request by bhumi46**
modified regclient install.sh ... added openssl creation as preinstall job

### CODE REVIEW

File: deployment/v3/mosip/regclient/install.sh [hunk 1]
```diff
@@ -55,7 +31,6 @@ function installing_regclient() {
     --set regclient.healthCheckUrl=$HEALTH_URL \
     --set regclient.hostName=$INTERNAL_HOST \
     --set istio.host=$REGCLIENT_HOST \
-    --wait \
```
Comment (syedsalman3753) [inline deployment/v3/mosip/regclient/install.sh:58]: Add `--wait --wait-for-jobs`

**Review by syedsalman3753 — CHANGES_REQUESTED** [review]

File: deployment/v3/mosip/regclient/install.sh (hunk 1 above)
Comment (bhumi46) [reply deployment/v3/mosip/regclient/install.sh]: added wait
there no wait command separatly for pre-install-jobs....wait is enough
the --wait flag will ensure that Helm waits for the completion of the pre-install job as well. This means:

The Helm install process will start.
The pre-install job will run.
Helm will wait for the pre-install job to complete.
After the pre-install job completes, Helm will continue with the rest of the chart installation.
Helm will then wait for all the other resources to be in a ready state.

**Review by syedsalman3753 — APPROVED** [review]

File: deployment/v3/mosip/regclient/README.md [hunk 2]
```diff
@@ -35,3 +35,7 @@ helm -n $NS install regclient mosip/regclient \
   
 ## Set up of regclient
 Follow the guide [here](https://github.com/mosip/registration-client/blob/develop/README.md)
+
+
+## Note:
```
Comment (ckm007) [inline deployment/v3/mosip/regclient/README.md:40]: remove note section

Comment (ckm007) [reply deployment/v3/mosip/regclient/README.md:40]: remove openssl from pre-requisites

Comment (bhumi46) [reply deployment/v3/mosip/regclient/README.md:40]: removed note and prerequisites

**Pull request closed without merging**